CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to Tart
+
+Table of Contents
+-----------------
+
+- [How to Build](#how-to-build)
+- [How to Create an Issue/Enhancement](#how-to-create-an-issueenhancement)
+- [Style Guidelines](#style-guidelines)
+- [Pull Requests](#Pull-Requests)
+
+## How to Build
+
+1. Fork the repository to your own GitHub account
+2. Clone the forked repository to your local machine
+3. If using Xcode, use from Xcode 15 beta 2 (Swift 5.9)
+4. In tart/scripts/run-signed.sh, find this line:
+```bash
+codesign --sign - --entitlements Resources/tart-dev.entitlements --force .build/debug/tart
+```
+5. Replace this line with your path values in the template literals: 
+ ```bash
+ codesign --sign - --entitlements ${TART_WORKSPACE_PATH}/Resources/tart-dev.entitlements --force ${BUILT_PRODUCTS_DIR}/tart"
+```
+6. In the same file, find this line:
+```bash
+.build/debug/tart "$@"
+```
+7. Replace with your build directory
+```bash
+ ${BUILT_PRODUCTS_DIR}/tart "$@"
+```
+8. Run the script to build and run tart
+```bash
+# usage: ./scripts/run-signed.sh run ventura-base
+```
+## How to Create an Issue/Enhancement
+
+1. Go to the [Issue page](https://github.com/cirruslabs/tart/issues) of the repository
+2. Click on the "New Issue" button
+3. Provide a descriptive title and detailed description of the issue or enhancement you're suggesting
+4. Submit the issue
+
+## Style Guidelines
+
+1. Code should follow camel case
+2. Code should follow [SwiftFormat](https://github.com/nicklockwood/SwiftFormat#swift-package-manager-plugin) guidelines
```

Comment:
```suggestion
2. Code should follow [SwiftFormat](https://github.com/nicklockwood/SwiftFormat#swift-package-manager-plugin) guidelines. You can auto-format the code by running the following command:


```bash
swift package plugin --allow-writing-to-package-directory swiftformat --cache ignore .
```
```